core/src/main/java/com/microsoft/applicationinsights/extensibility/initializer/SdkVersionContextInitializer.java

```diff
@@ -21,11 +21,9 @@
 
 package com.microsoft.applicationinsights.extensibility.initializer;
 
-import com.microsoft.applicationinsights.internal.logger.InternalLogger;
+import com.microsoft.applicationinsights.extensibility.ContextInitializer;
```

Comment:
I think this change is only about removing unused import. Why it's unnecessary? They are overhead.

Reply:
We can do style clean-up in a separate PR.

Reply:
I think there is no harm touching some files here. I agree big changes can go separately, this is just a import removal :)

Reply:
Sure. I'm fine with fixing some style issues here and there, but try to keep it contained to files which are related to the feature.

Reply:
That way, if we want a list of files having to do with this feature, we can simply look at files changed in this PR or the squashed commit. Having this here makes the git history look like this file has something to do with the spring boot starter.

Reply:
@littleaj I agree with you on this. I have tried to reduce majority of styling things at other places too.
#resolved

Reply:
This is not resolved.... please revert since it's has no relevance to this feature.

Reply:
@littleaj I thought we had discussed that since this was only an import removal, it was fine. If this opinion has changed and you think that we should keep the additional import then I can bring it back. In my opinion, we rarely get chance to work on code cleaning and small changes (which do not affect the logic should be fine). While I do understand your concern for clean history this should not be something which is too big of change. @grlima what are your thoughts? If you both agree on reverting this small change I can do that.

Reply:
@littleaj I have reverted this file. Thanks fore your concerns on separating the logic.